commons/src/main/java/io/aiven/kafka/connect/common/source/input/TransformationUtils.java

```diff
@@ -16,49 +16,19 @@
 
 package io.aiven.kafka.connect.common.source.input;
 
-import static io.aiven.kafka.connect.common.config.SchemaRegistryFragment.SCHEMA_REGISTRY_URL;
+import org.apache.kafka.connect.data.SchemaAndValue;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.lang.reflect.InvocationTargetException;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-
-import org.apache.kafka.common.config.AbstractConfig;
-
-import io.aiven.kafka.connect.common.config.SchemaRegistryFragment;
-
-import io.confluent.kafka.serializers.KafkaAvroSerializer;
+import io.confluent.connect.avro.AvroData;
 import org.apache.avro.generic.GenericRecord;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 final public class TransformationUtils {
-    private static final Logger LOGGER = LoggerFactory.getLogger(TransformationUtils.class);
+    private static final AvroData AVRO_DATA = new AvroData(100);
 
     private TransformationUtils() {
         // hidden
     }
 
-    public static byte[] serializeAvroRecordToBytes(final List<GenericRecord> avroRecords, final String topic,
-            final AbstractConfig sourceConfig) {
-        final SchemaRegistryFragment registryFragment = new SchemaRegistryFragment(sourceConfig);
-        final Map<String, String> config = Collections.singletonMap(SCHEMA_REGISTRY_URL,
-                registryFragment.getSchemaRegistryUrl());
-
-        try (KafkaAvroSerializer avroSerializer = (KafkaAvroSerializer) registryFragment.getAvroValueSerializer()
-                .getDeclaredConstructor()
-                .newInstance(); ByteArrayOutputStream out = new ByteArrayOutputStream()) {
-            avroSerializer.configure(config, false);
-            for (final GenericRecord avroRecord : avroRecords) {
-                out.write(avroSerializer.serialize(topic, avroRecord));
-            }
-            return out.toByteArray();
-        } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException
-                | IOException e) {
-            LOGGER.error("Error in reading s3 object stream for topic {} with error : {}", topic, e.getMessage(), e);
-        }
-        return new byte[0];
+    public static SchemaAndValue getValueStruct(final Object record) {
```

Comment:
Do we still need this? We can probably just fold it into the parquet/avro classes now as its pretty small.

Reply:
I agree, if I were looking for the meaning of this class, I would expect it to be in `AvroTransformer` (static, reused by  Parquet)  and take a `GenericRecord`  as a parameter!

Reply:
Removed.

Reply:
Moved :)